internal/network/command_dns_forwarder_update.go

```diff
@@ -28,13 +28,13 @@ func (c *command) newDnsForwarderUpdateCommand() *cobra.Command {
 	}
 
 	cmd.Flags().StringSlice("dns-server-ips", nil, "A comma-separated list of IP addresses for the DNS server.")
-	cmd.Flags().String("name", "", "Name of the DNS forwarder.")
 	cmd.Flags().StringSlice("domains", nil, "A comma-separated list of domains for the DNS forwarder to use.")
+	cmd.Flags().String("name", "", "Name of the DNS forwarder.")
```

Comment:
I feel like "name" should go first out of these three update flags. When we're printing the output of this command, I assume "name" (or maybe "id") appears first there too.

Reply:
updated flag order for help prompt (autocomplete is still alphabetical)